app/views/posts/new.html.erb

```diff
@@ -0,0 +1,17 @@
+<h1>Create a New Post</h1>
+
+<%= form_with(model: [@user, @post], local: true) do |form| %>
+  <div>
+    <%= form.label :title %>
+    <%= form.text_field :title %>
+  </div>
+
+  <div>
+    <%= form.label :text %>
+    <%= form.text_area :text %>
+  </div>
+
+  <div>
+    <%= form.submit "Create Post" %>
+  </div>
```

Comment:
- [ ] @ShiroYaksha90 Great work on your app, however, I was unable to test if the forms are working correctly, I got this error when trying to access the form page to create a new post, kindly consider fixing the error and ensure all forms are working s expected 
![image](https://github.com/ShiroYaksha90/blog-app/assets/61408860/1321e180-7a95-4216-a417-f5566bbd1a66)